### ISSUE TITLE
Add title to desktop page

### ISSUE THREAD

**Issue by sabuto** [kikipoulet/SukiUI#2]
I think it would be a good idea to be able to add a title to the custom desktop page like we can in traditional desktop apps, i'm happy to open a PR for this, the only issue i can see with this is the menu items but it may be worth putting them on a grid column below?

**Comment by kikipoulet** [comment]
I agree with the idea, moreover it would permit to use the DesktopPage with the sidemenu too.

I will do it this week because I'm incorporating an easy dialog system (using https://github.com/AvaloniaUtils/DialogHost.Avalonia ) in the DesktopPage, so I will have to update the documentation anyway 👍

**Comment by kikipoulet** [comment]
Done in the last commit, see the update readme

**Comment by sabuto** [comment]
sorry to be a pian, but what about the ability to set position of the title bar? for example personally i would have it centered.

**Comment by kikipoulet** [comment]
That's fine, I'm really open of making the library better so that's the process

We can do a TitleAlignment property + TitleFontSize + TitleFontWeight ?

**Comment by sabuto** [comment]
i agree :), also whilst i think about it, any perticular reason you use usercontrols for the custom controls instead of creating a control say like avalonia core does? im not 100% sure but i expect there is a reason why they do it that way.

**Comment by battistinidiego** [comment]
Hi, sorry, I missed some emails 😓

Could you summarize the situation in order to help us? Thank you very much sorry again

Diego Battistini

**Comment by sabuto** [comment]
Hey, no worries :)

so for example you have [CircleProgessBar](https://github.com/kikipoulet/SukiUI/blob/main/SukiUI/Controls/CircleProgressBar.axaml.cs) here with the properties but it is a user control. and then you have the styles defined in the xaml for that user control but avalonia itself makes controls like this [ProgessBar](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/ProgressBar.cs) which is not a user control, and then styles like this [ProgessStyle](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml) so it is contained in a style page. I am not sure of the implication but i would assume there is come benefit.

**Comment by kikipoulet** [comment]
new properties : 
- Title
- TitleHorizontalAlignment
- TitleFontSize
- TitleFontWeight

have been added in the new nuget package

---------------------------

About the UserControl/Control debate I did it just because of habits
But I don't think it makes a lot of changes anyway - from documentation : 

UserControl control is a Control that represents a reusable collection of controls in a predefined layout.
UserControl actually provides very little functionality on top of ContentControl. The difference is that you will not usually create instances of the UserControl class directly; instead a new subclass of the UserControl class is usually created for each "view" to be shown by an application.
